cypress/integration/Unit/details.spec.js

```diff
@@ -0,0 +1,24 @@
+/// <reference types="cypress" />
+import jwt from 'jsonwebtoken';
+describe('Documents', () => {
+  it('Has a link', () => {
```

Comment:
let's get a bit more creative and descriptive with our test names...  
```suggestion
  it('Title is a clickable link', () => {
```
or something like that